### PR TITLE
Documentation lesson to use markdown instead of Rst

### PR DESCRIPTION
**Pull request by samumantha**
The transformation is not yet fully finished and tested
Still to do:
* index.rst -> md to be done / found out if it works
* labeling math equations
* more testing
But I still suggest to merge these changes so that exercise leaders can see that we will use (at least mainly) markdown for the sphinx exercise and not rst.
I will still work on the to-dos before second week of the workshop.

What do you think?

### CODE REVIEW

File: content/sphinx.md [hunk 1]
```diff
@@ -323,42 +325,36 @@ the list of extensions in `conf.py`:
...
 ```
 
-An equation and equation array:
-```rst
-.. math::
-  :label: myequation
+An equation:
 
-  a^2 + b^2 = c^2
+:math:`a^2 + b^2 = c^2`
```
Comment (rkdarst) [inline content/sphinx.md:338]:
```suggestion
{math}`a^2 + b^2 = c^2`
```

File: content/sphinx.md [hunk 2]
```diff
@@ -323,42 +325,36 @@ the list of extensions in `conf.py`:
...
 
+or:
 .. math::
```
Comment (rkdarst) [inline content/sphinx.md:341]:
~~~
```{math}

File: content/sphinx.md [hunk 3]
```diff
@@ -323,42 +325,36 @@ the list of extensions in `conf.py`:
...
-  :label: myarray
-
-  \begin{eqnarray}
-    x^2 & : x < 0 \\
-    x^3 & : x \ge 0 \\
-  \end{eqnarray}
-```
+`a^2 + b^2 = c^2`
```
Comment (rkdarst) [inline content/sphinx.md:342]:
````suggestion
a^2 + b^2 = c^2
```
~~~
````

File: content/sphinx.md [hunk 4]
```diff
@@ -323,42 +325,36 @@ the list of extensions in `conf.py`:
...
 
 ---
 
 ### Where to find more
 
-- For more RST functionality, see the [Sphinx documentation](http://www.sphinx-doc.org/en/stable/rest.html)
-  and the [quick-reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html).
-- For Sphinx additions to standard RST, see [Sphinx Markup Constructs](http://www.sphinx-doc.org/en/1.7/markup/index.html).
+- For more Markdown functionality, see the [Mardown guide](https://www.markdownguide.org/basic-syntax/).
+- For Sphinx additions, see [Sphinx Markup Constructs](http://www.sphinx-doc.org/en/1.7/markup/index.html).
```
Comment (rkdarst) [inline content/sphinx.md:351]: Maybe we should clarify MD vs RSE vs MyST more.  But let's merge and I can try this on a second pass?

In particular linking to any MD guide except the MyST one will exclude all of the important Sphinx features - I guess this is a strategic problem here, the way the term "markdown" will become very confusing to learners.

File: content/sphinx.md [hunk 5]
```diff
@@ -258,52 +250,62 @@ Level 4
...
   }
+ ```
 ```
-- You could include the contents of an external file using `literalinclude` directive, as follows:
+- You could include the contents of an external file using `{include}` directive, as follows:
 
-```rst
-.. literalinclude:: filename
+```md
+ ```{include} ../README.md
```
[the remark this answers is not in the document]
Comment (rkdarst) [reply content/sphinx.md:290]: In this case yes, I think.  There is a MyST code block (one set), and inside of that is another MyST directive (more tildes)

**Comment by samumantha** [comment]
Please merge if you think it is ready and let's iterate then, so that exercise leaders (if they now check the exercises) get to see markdown instead of rst. (On phone only for the moment so cannot (dare not) merge myself 😅)

**Comment by bast** [comment]
Can somebody resolve the suggestions? It is not clear to me whether they have been applied or not.

File: content/sphinx.md (hunk 2 above)
Comment (rantahar) [reply content/sphinx.md:341]: This still needs to be fixed (but I can merge and create anoher PR)

**Pull request merged**